apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -52,10 +54,12 @@ const mapSingleItem = ({
       type: primaryCta.type,
       label: primaryCta.content,
       url: cta?.data.url,
+      isCompleted: actionType === ButtonTypeEnum.PRIMARY && actionStatus === MessageActionStatusEnum.DONE,
```

Comment:
We've been missing this information on the primary/secondary buttons, so I've added this field.